components/service/glean/src/main/java/mozilla/components/service/glean/storages/CountersStorageEngine.kt

```diff
@@ -30,6 +32,19 @@ internal open class CountersStorageEngineImplementation(
         }
     }
 
+    @Suppress("TooGenericExceptionCaught")
+    override fun serializeSingleMetric(
+        userPreferences: SharedPreferences.Editor?,
+        storeName: String,
+        value: Int
+    ) {
+        try {
+            userPreferences?.putInt(storeName, value)
```

Comment:
Looks like `putInt` [doesn't throw](https://developer.android.com/reference/android/content/SharedPreferences.Editor.html#putInt(java.lang.String,%20int)): let's get rid of the try/catch block (and also remove the `java.lang.Exception`.